Tests/Shared.Specs/EventAssertionSpecs.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if !NETCOREAPP1_1 && !NETSTANDARD1_3 && !NETSTANDARD1_6 && !NETSTANDARD2_0
+﻿#if !NETCOREAPP1_1 &&  !NETCOREAPP2_0 && !NETSTANDARD1_3 && !NETSTANDARD1_6 && !NETSTANDARD2_0 && !NETSTANDARD2_1
```

Comment:
Shouldn't `NETCOREAPP2_0` still be included?

Reply:
For some reason, it would complain this morning about "Monitor", now it doesn't. Fixed.